### PR TITLE
Flat mediation fee scenario

### PR DESCRIPTION
**Pull request by palango**
Part of #4748 

## Description

Adds scenarios for mediation fees

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

**Comment by palango** [comment]
@czepluch This scenario ran fine yesterday at my computer, I just ran into the fee limit. Now I increased the amount, so the fee limit should be fine. Can you run it locally again and check?

File: raiden/tests/scenarios/mfee1_flat_fee.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    pfs:
+      url: https://pfs-goerli-with-fee.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+
```
Comment (czepluch) [inline raiden/tests/scenarios/mfee1_flat_fee.yaml]: Could you please add `balance_min: 1_000_000` here then I think we can merge it.

Comment (palango) [reply raiden/tests/scenarios/mfee1_flat_fee.yaml]: Why do I need to do that? Shouldn't the SP mint enough tokens?

Comment (czepluch) [reply raiden/tests/scenarios/mfee1_flat_fee.yaml]: No, right now it's hard coded to only mint 5000 if nothing else is specified.

Comment (palango) [reply raiden/tests/scenarios/mfee1_flat_fee.yaml]: Ok, did the changes!

File: raiden/tests/scenarios/mfee1_flat_fee.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 100
+    proportional-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    proportional-imbalance-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+
+## This is the MFEE1 scenario. It creates a network with topology 0 -> 1 -> 2 -> 3 and checks
```
Comment (czepluch) [inline raiden/tests/scenarios/mfee1_flat_fee.yaml]: Could you also please add a similar description to the README?

**Pull request merged**